src/cpp/session/SessionMain.cpp

```diff
@@ -1701,7 +1702,7 @@ int saveWorkspaceAction()
 
 void syncRSaveAction()
 {
-   rstudio::r::session::setSaveAction(saveWorkspaceOption());
+   return r::session::setSaveAction(saveWorkspaceAction());
```

Comment:
Why was `return` added (the function has `void` return type)?

Reply:
No good reason, other than the esoteric feature that functions returning void can also `return functionThatReturnsVoid();` 🙂